server/src/main/java/org/elasticsearch/cluster/ClusterModule.java

```diff
@@ -91,9 +96,10 @@
 public class ClusterModule extends AbstractModule {
 
     public static final String BALANCED_ALLOCATOR = "balanced"; // default
+    public static final String DESIRED_BALANCE_ALLOCATOR = "desired_balance";
```

Comment:
NIT: we need to remove/move `default` comment

Reply:
Good point, addressed in 1b67865d429268724a32b5995ba3dee809e0a20e.